spec/features/disabled_get_an_identity_integration/happy_journeys/js/while_working_at_private_childcare_privider_but_not_a_nursery_spec.rb

```diff
@@ -62,9 +62,9 @@
 
     School.create!(urn: 100_000, name: "open manchester school", address_1: "street 1", town: "manchester", establishment_status_code: "1")
 
-    expect_page_to_have(path: "/registration/work-in-nursery", submit_form: true) do
-      expect(page).to have_text("Do you work in a nursery?")
-      page.choose("No", visible: :all)
+    expect_page_to_have(path: "/registration/kind-of-nursery", submit_form: true) do
+      expect(page).to have_text("Which early years setting do you work in?")
+      page.choose("Private nursery", visible: :all)
```

Comment:
hmm think this might have to be something else? @Lockyy

Reply:
What sort of error is this running into?